src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,42 @@
+import { SearchLanguage } from "./models";
+import { KeyboardData } from "./storage-models";
+
+/**
+ * @file KeymanAPI class to implement all services provided by official Keyman API
+ * Note: Currently this class is used for the functionality to fetch all existing keyboard languages
+ * and more function(s) could be added in the future
+ */
+export class KeymanAPI {
+  /**
+   * API configs
+   */
+  baseUrl = "https://api.keyman.com/search";
+  /**
+   * Query all languages data, doc: https://help.keyman.com/developer/cloud/search/1.0/
+   */
+  params = { q: "" };
+  url: URL;
+  languages: KeyboardData[];
```

Comment:
My issue with this is why is this an instance property? What purpose does it serve as an instance property? Why _must_ it be more than a local variable in the scope of `fetchLanguageData()`?